packages/rancher-istio/charts/values.yaml

```diff
@@ -1,11 +1,19 @@
 overlayFile: ""
 tag: 1.10.4
-##Setting forceInstall: true will remove the check for istio version < 1.6.x and will not analyze your install cluster prior to install
-forceInstall: false
 
 installer:
-  repository: rancher/istio-installer
-  tag: 1.10.4-rancher1
+  repository: brendarearden/istio-test
```

Comment:
Should we use a rancher's repo here?

Reply:
Yes, I will switch this over once the new image is tagged .